alacritty/src/cursor.rs

```diff
@@ -15,10 +13,12 @@ pub fn get_cursor_glyph(
     cursor_thickness: f64,
 ) -> RasterizedGlyph {
     // Calculate the cell metrics.
-    let height = metrics.line_height as i32 + i32::from(offset_y);
-    let mut width = metrics.average_advance as i32 + i32::from(offset_x);
-
-    let line_width = cmp::max((cursor_thickness * f64::from(width)).round() as i32, 1);
```

Comment:
Just a small thing, but I personally have a much better time reading `max(x, y)` than I do `x.max(y)`. But it's not a problem if you really want to change it.

Reply:
> Just a small thing, but I personally have a much better time reading max(x, y) than I do x.max(y). But it's not a problem if you really want to change it.

While I understand the sentiment, since I personally was confused the first time I read it, I'm not sure we want to set this as a policy. Generally methods do what they're documented to do, which is exactly what `.max()` does. It is also a shorthand that can reduce complexity when you know already what it does, since it is a good chunk shorter and takes just a single value. It's also used in multiple places in Alacritty already (including this file).

So I'd generally be against forcing the usage of `max(x, y)` over `x.max(y)`.